package.json

```diff
@@ -95,16 +95,16 @@
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-15": "^1.0.5",
     "enzyme-to-json": "^3.3.0",
-    "eslint": "^5.0.1",
-    "eslint-config-airbnb": "^16.1.0",
+    "eslint": "^4.19.1",
```

Comment:
Why has eslint's version dropped from 5.x to 4.x?

Reply:
Because `eslint-config-airbnb` has a peer on 4.19.1 and hasn't upgraded to 5 yet - we prematurely upgraded to 5 in a PR the other day.

Reply:
Ahh gotcha